app/src/main/java/org/oppia/app/home/RouteToExplorationListener.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.app.home
+
+/** Listener for when an activity should route to a exploration. */
+interface RouteToExplorationListener {
+  fun routeToExploration()
```

Comment:
Shouldn't this also take the exploration ID? I feel like the exploration activity will need it (for e.g. to display the title now, and later displaying author information, etc.)

Reply:
Okay got it. We can use this exploration id to get other details related to the exploration.

Reply:
Updated code to transfer explorationId too.